NEWS.md

```diff
@@ -1,3 +1,10 @@
+## 2.0.5 (2020-11-11)
+
+
+### Bug Fixes
+
+- Fix bug where interpolations were unnecessarily resolved during merge ([#431](https://github.com/omry/omegaconf/issues/431))
+
```

Comment:
Please clean you diff not include changes from other diffs. (rebase, don't merge).